addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4456,7 +4457,7 @@ const ImageHandlerOption = SnippetOptionWidget.extend({
      */
     async _renderCustomXML(uiFragment) {
         const img = this._getImg();
-        if (!this.originalSrc || !['image/png', 'image/jpeg'].includes(this._getImageMimetype(img))) {
```

Comment:
I see your method now accepts BMP, what this not the case before? :thinking:

Reply:
Actually, after checking the code, it seems that [BMP images are converted to PNG](https://github.com/odoo/odoo/blob/b3a129395d0823bce8cf2a3f286f4dd10c98a316/odoo/tools/image.py#L124) in `ImageProcess > image_base64()` so I think there is no need to use this in the filter.